ci-operator/config/konveyor/mig-operator/konveyor-mig-operator-master.yaml

```diff
@@ -0,0 +1,60 @@
+base_images:
+  base:
+    name: "4.8"
+    namespace: ocp
+    tag: base
+  mig-operator:
+    name: "4.8"
+    namespace: ocp
+    tag: mig-operator
+binary_build_commands: make build
+build_root:
+  use_build_cache: true
+  project_image:
+    dockerfile_path: build/Dockerfile
+images:
+  - dockerfile_path: build/Dockerfile
```

Comment:
Why are we building two images from the same dockerfile with different bases?

Reply:
Ahh, removng it